build.cake

```diff
@@ -139,6 +139,25 @@ Task("PackageZip")
     Zip(BIN_DIR, File(path), zipFiles);
 });
 
+Task("PackageChocolatey")
+.Does(() =>
+{
+	CreateDirectory("PACKAGE_DIR");
+
+	ChocolateyPack("nunit-project-editor.nuspec", 
+		new ChocolateyPackSettings()
+		{
+			OutputDirectory = PACKAGE_DIR,
+			Files = new []
+			{
+				new ChocolateyNuSpecContent { Source = "LICENSE.txt" },
+				new ChocolateyNuSpecContent { Source = "CHANGES.txt" },
+				new ChocolateyNuSpecContent { Source = BIN_DIR + "nunit-editor.exe", Target = "bin" },
```

Comment:
I'm not sure I understand this. The docs indicate it should be done when the package downloads an installer from a url using http: or https:. This package contains the binaries themselves rather than a setup file. 

I suppose one approach would be for you to compare a checksum over the entire nupkg that's uploaded to chocolatey to the same nupkg as downloaded from github or from our myget feed.

Do you have any example of a package that's structured in the same way and how that gets verified?